wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/ClusterTopologyMonitorImpl.java

```diff
@@ -265,8 +256,13 @@ protected List<HostSpec> waitTillTopologyGetsUpdated(final long timeoutMs) throw
     return latestHosts;
   }
 
+  private List<HostSpec> getStoredHosts() {
+    Topology topology = storageService.get(Topology.class, this.clusterId);
+    return topology == null ? null : topology.getHosts();
+  }
+
   @Override
-  public void close() throws Exception {
+  public void stop() {
```

Comment:
Should we relocate it to AbstractMonitor?

Reply:
I've now refactored things so that there is a default stop() implementation in AbstractMonitor that concrete implementations can override if needed